ui/redux/rootReducer.js

```diff
@@ -86,6 +86,19 @@ export const loadUserOptions = () => {
   }
 }
 
+export const loadStaffOptions = () => {
+  return (dispatch) => {
+    dispatch({ type: REQUEST_USERS })
+    new HttpRequestHelper('/api/users/get_all_staff',
```

Comment:
I notice you aren't checking if these are already loaded before deciding whether or not to request these. Is the bug we were seeing related to that fixed?

Reply:
Yes, it is fixed now.